pedsnet/v4.5/models.csv

```diff
@@ -0,0 +1,2 @@
+model,version,release_level,release_serial,label,url,description
+pedsnet,v4.5,alpha,3,PEDSnet v4.4,http://pedsnet.info,
```

Comment:
@boss-sam Just a couple of changes here:
- release_serial: change to 0
- label: change to PEDSnet v4.5